### PR TITLE
[refactor](jdbc) refactor jdbc connection num in datasource

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes
now maybe jdbc have problem that there are too many connections  and they do not release,
so change the property of datasource: init = 1, min = 1, max = 100, and idle time is 10 minutes.

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by BePPPower — COMMENTED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java [hunk 1]
```diff
@@ -98,8 +98,10 @@ public JdbcClient(String user, String password, String jdbcUrl, String driverUrl
             dataSource.setUsername(jdbcUser);
             dataSource.setPassword(password);
             dataSource.setMinIdle(1);
-            dataSource.setInitialSize(2);
-            dataSource.setMaxActive(5);
+            dataSource.setInitialSize(1);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java:101]: Change these variables to fe.conf and be.conf

**Comment by zhangstar333** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**